sdk/communication/Azure.Communication.CallingServer/src/Models/CallConnectionState.cs

```diff
@@ -8,8 +8,8 @@ namespace Azure.Communication.CallingServer
     /// <summary>
     /// The states of a call.
     /// </summary>
-    [CodeGenModel("CallState", Usage = new string[] { "input", "output" }, Formats = new string[] { "json" })]
-    public readonly partial struct CallState
+    [CodeGenModel("CallConnectionState", Usage = new string[] { "input", "output" }, Formats = new string[] { "json" })]
+    public readonly partial struct CallConnectionState
```

Comment:
Should we name this `CallConnectionStatus` instead of `CallConnectionState`. We have a call state in the calling client that seems to be different. This seems more of a status than a state.

Reply:
Status could be use if are referring to whether call is currently active or not, but state give its current stage in whole overall call process.

Calling client is using different verbs but meaning of some of them are same.

Reply:
> Calling client is using different verbs but meaning of some of them are same.

But should not we use the same terminology?

Reply:
Naveed changed the meaning to be the same as client SDK - connecting/connected/disconnecting/disconnected